src/artifact.c

```diff
@@ -4699,8 +4699,12 @@ int dieroll; /* needed for Magicbane and vorpal blades */
 	}
 
 	if(otmp->otyp == PINCER_STAFF && u.uinsight >= 10){
-		if(otmp->ovar1_pincerTarget == mdef->m_id)
+		if(otmp->ovar1_pincerTarget == mdef->m_id){
 			*plusdmgptr += basedmg;
+			if (u.uinsight >= 20 && otmp->oartifact && otmp->oartifact == ART_FALLINGSTAR_MANDIBLES && !Magic_res(mdef)){
+				*truedmgptr += reduce_dmg(mdef, d(1, 12), FALSE, TRUE);
```

Comment:
Does it make sense to call reduce_dmg for this dmg? What does reduce_dmg do? I don't know, I'm just asking.

Reply:
I assume this is for checking half spell dam maybe?

Reply:
yeah, it's for half spell/phys dam (spell here). see above lines about like, diskos extra damage & such